src/main/java/cn/nukkit/entity/projectile/EntityThrownTrident.java

```diff
@@ -207,18 +208,7 @@ public Entity create(Object type, Position source, Object... args) {
         FullChunk chunk = source.getLevel().getChunk((int) source.x >> 4, (int) source.z >> 4);
         if (chunk == null) return null;
 
-        CompoundTag nbt = new CompoundTag()
-                .putList(new ListTag<DoubleTag>("Pos")
-                        .add(new DoubleTag("", source.x + 0.5))
-                        .add(new DoubleTag("", source.y))
-                        .add(new DoubleTag("", source.z + 0.5)))
-                .putList(new ListTag<DoubleTag>("Motion")
-                        .add(new DoubleTag("", 0))
-                        .add(new DoubleTag("", 0))
-                        .add(new DoubleTag("", 0)))
-                .putList(new ListTag<FloatTag>("Rotation")
-                        .add(new FloatTag("", new Random().nextFloat() * 360))
-                        .add(new FloatTag("", 0)));
+        CompoundTag nbt = Entity.getDefaultNBT(source.add(0.5, 0, 0.5), new Vector3(), new Random().nextFloat() * 360, 0);
```

Comment:
`new Vector3()` is not necessary, the param accepts `null` and sets x y z to `0`